packages/markup-frontstage/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "@bentley/markup-frontstage-react",
+  "version": "1.0.0",
+  "description": "Markup Frontstage React",
+  "license": "MIT",
+  "main": "lib/markup-frontstage-react.js",
+  "typings": "lib/markup-frontstage-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize && npm run copy-css",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets && npm run copy-css",
+    "clean": "rimraf lib build",
+    "lint": "tslint -p . 1>&2",
+    "test": "jest",
+    "copy-css": "copyfiles -u1 \"./src/**/*.scss\" ./lib",
+    "cover": "npm test"
+  },
+  "keywords": [
+    "Bentley",
+    "iModel Markup Frontstage React",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/packages/markup-frontstage"
+  },
+  "dependencies": {
+    "classnames": "^2.2.6",
+    "i18next": "^10.2.2",
+    "i18next-browser-languagedetector": "^4.0.2"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-transform-runtime": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
+    "@babel/preset-react": "^7.9.4",
+    "@babel/preset-typescript": "7.9.0",
+    "@bentley/bentleyjs-core": "^2.0.0",
+    "@bentley/build-tools": "^2.0.0",
+    "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@bentley/imodeljs-i18n": "^2.0.0",
+    "@bentley/imodeljs-markup": "^2.0.0",
+    "@bentley/presentation-frontend": "^2.0.0",
+    "@bentley/ui-abstract": "^2.0.0",
+    "@bentley/ui-components": "^2.0.0",
+    "@bentley/ui-core": "^2.0.0",
+    "@bentley/ui-framework": "^2.0.0",
+    "@testing-library/react": "^10.4.5",
+    "@types/classnames": "^2.2.3",
+    "@types/i18next": "^8.4.2",
+    "@types/jest": "^24.0.15",
+    "@types/react": "16.9.23",
+    "@types/react-dom": "16.9.5",
+    "@types/redux-mock-store": "^1.0.2",
+    "babel-jest": "^24.9.0",
+    "babel-plugin-import-remove-resource-query": "^1.0.0",
+    "concurrently": "^4.1.0",
+    "copyfiles": "^2.1.0",
+    "cpx": "^1.5.0",
```

Comment:
i recommend either using cpx or copyfiles, really no need for both. scripts should be updated accordingly

Reply:
completely forgot to remove it while I and Jason were trying to fix css problem, thanks, removed now!